### ISSUE TITLE
bundle css doesnt handle @import correct

### ISSUE THREAD

**Issue by puschie286** [abpframework/abp#17820]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

when using @import in an css file that is bundled its just minified but not moved to the top

### Reproduction Steps

1. create css file that contain import statement
2. add this css file to a bundle as second entry
3. create release build, start and see generated bundle file

### Expected behavior

all import statements should be minified and moved to top of the generated bundle file

### Actual behavior

import statement is only minified

### Version

v6.0

### User Interface

Common (Default)

### Database Provider

EF Core (Default)

### Tiered or separate authentication server

None (Default)

### Operation System

Windows (Default)

**Comment by maliming** [comment]
hi

Please share the detailed steps, Thanks.

**Comment by puschie286** [comment]
@maliming use `@import` in an css file that is part of a bundle - the reproduce steps are part of the issue. if you have problems reproduce it, please state what the problem is

**Comment by maliming** [comment]
Can you share a template project on GitHub?

**Comment by puschie286** [comment]
@maliming sorry but i dont understand what the problem is - the issue should be very clear and straight forward, if something is unclear, please state the problem

[documentation](https://docs.abp.io/en/abp/latest/UI/AspNetCore/Bundling-Minification#creating-a-new-bundle)

**Comment by maliming** [comment]
hi

You can create a brand new template project, apply your `Reproduction Steps`, and then share it on GitHub. I will download it to check it.

**Comment by puschie286** [comment]
@maliming why don’t you do it ?

**Comment by rqx110** [comment]
if we have two css:
1.css
```css
@import "custom1.css";
.red { color:red}
```

2.css
```css
@import "custom2.css";
.yellow {color: yellow}
```

after abp bundle, it will create with
bundle.css
```css
@import "custom1.css";
.red { color:red}
@import "custom2.css";
.yellow {color: yellow}
```
the issue is the second `import` should on the top of file.
@puschie286, does i explain right?

**Comment by puschie286** [comment]
yes - from the [documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/@import?retiredLocale=de) : An @import rule must be defined at the top of the stylesheet, before any other at-rule (except [@charset](https://developer.mozilla.org/en-US/docs/Web/CSS/@charset) and [@layer](https://developer.mozilla.org/en-US/docs/Web/CSS/@layer)) and style declarations, or it will be ignored.

**Comment by maliming** [comment]
The bundle is just merging and compressing javascript and css files. That's all.

**Comment by puschie286** [comment]
Some css features (like import) require a reordering. This is especialy problematic when you want to include some libary css that might contain import statements or have added an import statement in the future updates.

You could create seperated bundles for each libary/file you want to include that contain import statements, but you will lose the possibility to override rules by order ([Cascading Order, 4.](https://developer.mozilla.org/en-US/docs/Web/CSS/Cascade#cascading_order))

(For now) the documentation should mention that import statements does not work ( or only work for the first css file ) or implement some simple reordering of import statements.

implementation should be relative simple:
1. search for import statements and move in list (remove from merged file)
2. update import path (relative to the origin css file)
3. add import list add the beginning of the merged file
